src/lib/mathlib/math/Limits.hpp

```diff
@@ -56,6 +56,12 @@ constexpr _Tp min(_Tp a, _Tp b)
 	return (a < b) ? a : b;
 }
 
+template<typename _Tp>
+constexpr _Tp min(_Tp a, _Tp b, _Tp c)
```

Comment:
Same for max?

Reply:
I was unaware there was a need even before my change. Found multiple `min(min())` `max(max())` and replaced them in b58869b